config/software/openssl.rb

```diff
@@ -23,7 +23,7 @@
 dependency "cacerts"
 dependency "openssl-fips" if fips_mode?
 
-default_version "1.0.2y"
+default_version "1.1.1k"
```

Comment:
This needs to stay 1.0.2 until a FIPS enabled build past 1.0.2 comes out

Reply:
Sure @tas50